src/HPRT/Analytics/views.py

```diff
@@ -66,6 +70,49 @@ class AnalyticResultsView(LoginRequiredMixin, ListView):
     redirect_field_name = 'redirect_to'
     model = Patient
 
+    @staticmethod
+    def get_patient_data(data_list):
```

Comment:
Can you just add a one line description about what this function does? Parameter/what it's returning